### PR TITLE
[music3] Histogram - medClutEditorToolBox add action in view tools

### PR DESCRIPTION
**Pull request by mathildemerle**
Partially from https://github.com/Inria-Asclepios/medInria-public/pull/539

This PR adds in the views "Tools" button a "Histogram" button. It opens and closes an histogram in the view. The histogram can be added to toolboxes.

:m:

### CODE REVIEW

**Comment by mathildemerle** [comment]
Rebased PR

File: src/layers/legacy/medCoreLegacy/gui/viewContainers/medViewContainer.cpp [hunk 1]
```diff
@@ -74,7 +74,7 @@ class medViewContainerPrivate
     QAction* vSplitAction;
     QAction* hSplitAction;
     QPushButton* closeContainerButton;
-
+    QAction* histogramAction;
```
Comment (Florent2305) [inline src/layers/legacy/medCoreLegacy/gui/viewContainers/medViewContainer.cpp]: Ok but need rebase

File: src/plugins/legacy/medVtkView/medVtkView.cpp [hunk 2]
```diff
@@ -38,6 +38,7 @@
 #include <vtkMath.h>
 #include <vtkMetaDataSet.h>
 
+#include <medClutEditorToolBox.h>
```
Comment (Florent2305) [inline src/plugins/legacy/medVtkView/medVtkView.cpp]: Ok but need rebase

**Comment by mathildemerle** [comment]
This PR has been rebased

**Pull request merged**